aspnetcore/signalr/introduction-signalr-core.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Introduction to SignalR on ASP.NET Core
+author: rachelappel
+description: Learn how the ASP.NET Core SignalR library simplifies adding real-time web functionality to apps.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 2/19/2018
+ms.prod: aspnet-core
+ms.technology: dotnet-signalr
+ms.topic: article
+uid: signalr/introduction-signalr-core
+---
+# Introduction to SignalR
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+## What is SignalR?
+
+ASP.NET Core SignalR is a library that simplifies adding real-time web functionality to apps. Real-time web functionality is the ability to have server code push content to connected clients instantly as it becomes available, rather than having the server wait for a client to request new data.
+
+Good candidates for adding real-time updates include:
+
+* Apps that require high frequency updates from the server. Real-time gaming, social networks, maps and GPS apps, require high frequency updates.
+* Dashboards and monitoring apps. Some examples of these include voting and auction software.
+* Collaborative apps. Whiteboard apps and team meeting software are examples of collaborative apps.
+* Apps that require notifications. Social networks, email, chat, games, and many more types of apps use notifications.
+
+SignalR provides an API for creating server-to-client remote procedure calls (RPC). The RPCs call JavaScript functions on browsers or other client platforms from server-side .NET Core code. The SignalR API includes features such as connection management, groups,  messaging, and security.
+
+SignalR for ASP.NET Core:
+
+* Handles connection management automatically.
+* Enables broadcasting messages to all connected clients simultaneously. For example, a chat room.
+* Enables sending messages to specific clients or groups of clients.
+* Is open-sourced at [GitHub](https://github.com/aspnet/SignalR).
+* Scales nicely.
+
+The connection between the client and server is persistent, unlike an HTTP connection. 
+
+## Transports and fallbacks
+
+ SignalR is an abstraction over some of the transports that are required to do real-time work between client and server. A SignalR connection starts as HTTP. SignalR uses the [WebSocket transport](https://tools.ietf.org/html/rfc7118) when available, and falls back to other transports when it's not available. WebSocket is the ideal transport for SignalR. This is because it makes the most efficient use of server memory, has the lowest latency, and has the most underlying features, such as full duplex communication between client and server. SignalR automatically assigns a transport per connection for you automatically, though you can override this functionality.
```

Comment:
> This is because it makes the most efficient use of server memory, has the lowest latency, and has the most underlying features, such as f

Too long. Also avoid This
Perhaps 
WebSocket is the ideal transport for SignalR:

* it makes the most efficient use of server memory.
* etc